### DISCUSSION TITLE
组件的 style 的类型要加any吗？

### DISCUSSION THREAD

**Discussion by TinsFox** [3lang3/react-vant#536] [Q&A]
https://github.com/3lang3/react-vant/blob/9dee45dd58776131a1cc552adc6e89f127e3f51e/packages/react-vant/src/components/utils/interface.ts#L4

组件的 style 的类型要加any吗？
1. 现在使用的时候加style，ts的提示丢失了
2. antd的写法是没有any的，单纯是`style?:React.CSSProperties`

**Comment by 3lang3** [comment]
> https://github.com/3lang3/react-vant/blob/9dee45dd58776131a1cc552adc6e89f127e3f51e/packages/react-vant/src/components/utils/interface.ts#L4
> 
> 组件的 style 的类型要加any吗？
> 1. 现在使用的时候加style，ts的提示丢失了
> 2. antd的写法是没有any的，单纯是`style?:React.CSSProperties`

可以优化，这里偷懒了，为了在style里直接写 css var
